### ISSUE TITLE
Login fails if target file is empty

### ISSUE THREAD

**Issue by zaneb** [openshift/oc#985]
Since #893, the command `oc registry login --kubeconfig=aaa --to=bbb` fails if the file `bbb` is empty:

```
+(utils.sh:530): write_pull_secret(): oc registry login --kubeconfig=/tmp/kubeconfig--0dZLmzoYmT --to=/tmp/pullsecret--Sm82grIvD0
info: Using registry public hostname registry.ci.openshift.org
error: 1 error occurred:
	* reading JSON file "/tmp/pullsecret--Sm82grIvD0": unmarshaling JSON at "/tmp/pullsecret--Sm82grIvD0": unexpected end of JSON input
```

**Comment by atiratree** [comment]
being solved in #1011

**Comment by atiratree** [comment]
@zaneb the regression should be fixed but nevertheless this behaviour is getting deprecated so It is advisable to keep your fix openshift-metal3/dev-scripts#1318 in
